src/test/lro-poller.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { expect } from "chai";
+import * as nock from "nock";
+
+import { LroPoller, LroPollerOptions } from "../lro-poller";
+import TimeoutError from "../throttler/errors/timeout-error";
+
+const TEST_ORIGIN = "https://firebasedummy.test.com";
```

Comment:
`test.com` is an actual website not owned by Google, so let's just use `googleapis.com`

Reply:
Fixed